### PR TITLE
generate uuid function

### PR DESCRIPTION
**Pull request by cazterk**
- [x] create standalone uuid function

### CODE REVIEW

File: itez/beneficiary/uuid.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import uuid
+
+
+def generate_id():
+    """
+     this function generate a 7 digit code for agent
+    """
+    uuid_one = uuid.uuid4()
+    print(uuid_one)
```
Comment (sonlinux) [inline itez/beneficiary/uuid.py]: we dont  need to print  it anywhere @cazterk 
lets get rid of this

File: itez/beneficiary/uuid.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import uuid
+
+
+def generate_id():
+    """
+     this function generate a 7 digit code for agent
```
Comment (sonlinux) [inline itez/beneficiary/uuid.py]: The docs sentence must be capitalized like a regular sentence and end with a fullstop

File: itez/beneficiary/uuid.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import uuid
+
+
+def generate_id():
```
Comment (sonlinux) [inline itez/beneficiary/uuid.py]: I would rather call this `generate_uuid_and_agent_code():`

the name needs to tell enough  context of what the function does

File: itez/beneficiary/uuid.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    """
+     this function generate a 7 digit code for agent
+    """
+    uuid_one = uuid.uuid4()
+    print(uuid_one)
+    agent_code = str(uuid_one)[1:8].upper()
+    return (agent_code)
```
Comment (sonlinux) [inline itez/beneficiary/uuid.py]: we are looking to have both the uuid and agent code from this utitility
why are we returning on the  agent code?

I thought I wrote demo code  for you already?

**Review by sonlinux — CHANGES_REQUESTED** [review]

File: itez/beneficiary/uuid.py (hunk 1 above)
Comment (cazterk) [reply itez/beneficiary/uuid.py]: 'print,' removed

File: itez/beneficiary/uuid.py (hunk 2 above)
Comment (cazterk) [reply itez/beneficiary/uuid.py]: grammar corrected

File: itez/beneficiary/uuid.py (hunk 3 above)
Comment (cazterk) [reply itez/beneficiary/uuid.py]: renamed to => generate_uuid_and_agent_code():

File: itez/beneficiary/uuid.py (hunk 4 above)
Comment (cazterk) [reply itez/beneficiary/uuid.py]: now returning uuid as well

File: itez/beneficiary/uuid.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import uuid
+
+
+def generate_uuid_and_agent_code():
+    """
+     This function generate a 7 digit code for agent.
+    """
+    uuid_one = uuid.uuid4()
+    agent_code = str(uuid_one)[1:8].upper()
```
Comment (sonlinux) [inline itez/beneficiary/uuid.py]: for this lets get the mids values between the `-something found here-` not the first 7

**Review by sonlinux — CHANGES_REQUESTED** [review]

File: itez/beneficiary/uuid.py (hunk 5 above)
Comment (sonlinux) [reply itez/beneficiary/uuid.py]: also move the uuid utility from `uuid.py` to `utils.py`

Comment (cazterk) [reply itez/beneficiary/uuid.py]: moved uuid.py to utils.py

**Review by sonlinux — APPROVED** [review]
LGTM

Thank you @cazterk

**Pull request merged**